### PR TITLE
LPS-70129 Apply comparator

### PR DESCRIPTION
**Pull request by leoadb**

### CODE REVIEW

**Comment by leoadb** [comment]
cc @marcellustavares

File: modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/java/com/liferay/dynamic/data/lists/form/web/internal/portlet/action/GetDataProvidersMVCResourceCommand.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+ */
+@Component(
+	immediate = true,
+	property = {
+		"javax.portlet.name=" + DDLFormPortletKeys.DYNAMIC_DATA_LISTS_FORM_ADMIN,
+		"mvc.command.name=getDataProviders"
+	},
+	service = MVCResourceCommand.class
+)
+public class GetDataProvidersMVCResourceCommand extends BaseMVCResourceCommand {
```
Comment (marcellustavares) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/java/com/liferay/dynamic/data/lists/form/web/internal/portlet/action/GetDataProvidersMVCResourceCommand.java]: @leoadb please send a follow up renaming this class to `GetDataProviderInstancesMVCResourceCommand`

File: modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/java/com/liferay/dynamic/data/lists/form/web/internal/portlet/action/GetDataProvidersMVCResourceCommand.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Rafael Praxedes
+ */
+@Component(
+	immediate = true,
+	property = {
+		"javax.portlet.name=" + DDLFormPortletKeys.DYNAMIC_DATA_LISTS_FORM_ADMIN,
+		"mvc.command.name=getDataProviders"
```
Comment (marcellustavares) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/java/com/liferay/dynamic/data/lists/form/web/internal/portlet/action/GetDataProvidersMVCResourceCommand.java]: Please use `getDataProviderInstances`

File: modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/java/com/liferay/dynamic/data/lists/form/web/internal/portlet/action/GetDataProvidersMVCResourceCommand.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+		Locale locale = themeDisplay.getLocale();
+		long[] groupIds = _portal.getCurrentAndAncestorSiteGroupIds(
+			themeDisplay.getScopeGroupId());
+
+		List<DDMDataProviderInstance> ddmDataProviderInstances =
+			_ddmDataProviderInstanceLocalService.getDataProviderInstances(
+				groupIds, start, end);
+
+		JSONArray dataProvidersJSONArray = _jsonFactory.createJSONArray();
```
Comment (marcellustavares) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/java/com/liferay/dynamic/data/lists/form/web/internal/portlet/action/GetDataProvidersMVCResourceCommand.java]: Please rename to `dataProviderInstancesJSONArray`

File: modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/java/com/liferay/dynamic/data/lists/form/web/internal/portlet/action/GetDataProvidersMVCResourceCommand.java [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+		for (DDMDataProviderInstance ddmDataProviderInstance :
+				ddmDataProviderInstances) {
+			JSONObject dataProviderJSONObject = _jsonFactory.createJSONObject();
```
Comment (marcellustavares) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/java/com/liferay/dynamic/data/lists/form/web/internal/portlet/action/GetDataProvidersMVCResourceCommand.java]: Rename to `dataProviderInstanceJSONObject`

**Comment by marcellustavares** [comment]
@leoadb, I've added a few comments, please send a follow up on this.

Thx

**Comment by brianchandotcom** [comment]
@leoadb can you review changes from @marcellustavares and then resend? Thx!

**Comment by rafaprax** [comment]
resent at https://github.com/brianchandotcom/liferay-portal/pull/46241

**Pull request closed without merging**